### ISSUE TITLE
bug: Libraries are missing randomly

### ISSUE THREAD

**Issue by kevincaradant** [cnpm/npminstall#31]
I think i met some troubles with the dependancies and it's random, even after the upgrade in 1.0.8. 

If you try `npminstall`  with this repository => `https://github.com/kevincaradant/web-template` and then you try to build with the command `npm run prod`. Libraries are missing randomly. Sometimes, it's imagemin-pngquant, or 'recess'.

Sometimes that work the first time and sometimes not :/. Did you have any idea why ? 

Thank you 

![image](https://cloud.githubusercontent.com/assets/6093175/13380983/2b3af566-de50-11e5-99a7-631bdbf6538f.png)

And this module is in the package.json.

**Comment by kevincaradant** [comment]
ok i found my problem. In fact,  `npm install XXXX` , will order the package json by alphabetical order and for example, `gulp-plumber` need `squeak module` to the postinstall but your npminstall says me , `cannot find module squeak module` even if i had it on the package module because `squeak module` is at the end of the dev dependancies list ... so if i run again npminstall , it works or also, if i put `squeak module` before `gulp-plumber` in the dev-dependancies list.  Do you think you can fix this bug or not ?

**Comment by dead-horse** [comment]
I think it's the same issue with #30 ?

can you provide your `node` / `gulp` / `npminstall` versions?

**Comment by fengmk2** [comment]
@kevincaradant Please upgrade to npminstall@1.1.0 and retry again.

**Comment by kevincaradant** [comment]
NODE V5.7.0
NPM V3.6.0
NPMINSTALL V1.1.0

And my problem is not fix. Look this :

![image](https://cloud.githubusercontent.com/assets/6093175/13401206/863c9a6c-df0b-11e5-8adf-8072fa1dfb5a.png)


I don't want to put an under dependencie of gulp-plumber in my dev dependancies list in the package.json but, to try, even if i add  `squeak` module in the dev dependancie list. `squeak module` is at the end of the list, so npminstall, install it more later (too late) for gulp-plumber which haven't squeak when it wants to use it.  

You can see `squeak module` here, almost at the end  : 

![image](https://cloud.githubusercontent.com/assets/6093175/13401281/d0eff18a-df0b-11e5-9f3a-6dd8b5cfaf86.png)

**Comment by dead-horse** [comment]
I think https://github.com/cnpm/npminstall/pull/34 can fix this problem, wait @fengmk2 review and release a new version.

**Comment by kevincaradant** [comment]
@dead-horse , i tried your pach in the await branch. That resolve the problem about the order of dependancies so +1 :)

BUT, there are always the problem about under dependencies not found if i don't add them in the dev-dependencies :/.

Example : 
![image](https://cloud.githubusercontent.com/assets/6093175/13406214/8d9466d4-df22-11e5-9350-3fd5d79c9eef.png)

**Comment by dead-horse** [comment]
Interesting. I can't reproduce this problem in my laptop, can you provide the completed install log? Looks like it's cause by the same reason. You can try `npminstall@1.1.1` to see if it was fixed or not.

**Comment by kevincaradant** [comment]
First install No problem. 
I deleted node_module and try again and the problem appeared again : 

This is my log :

```
[angular-sanitize@1.5.0] installed at node_modules/.npminstall/angular-sanitize/1.5.0/angular-sanitize (0 packages, use 396ms, speed 22.1kB/s, json 10.06kB, tarball 0B)
[angular-mocks@1.5.0] installed at node_modules/.npminstall/angular-mocks/1.5.0/angular-mocks (0 packages, use 414ms, speed 21.44kB/s, json 10.06kB, tarball 0B)
[angular-animate@1.5.0] installed at node_modules/.npminstall/angular-animate/1.5.0/angular-animate (0 packages, use 433ms, speed 20.86kB/s, json 10.06kB, tarball 0B)
[angular-translate-loader-static-files@2.9.2] installed at node_modules/.npminstall/angular-translate-loader-static-files/2.9.2/angular-translate-loader-static-files (0 packages, use 802ms, speed 33.22kB/s, json 28.67kB, tarball 0B)
[angular@1.5.0] existed at node_modules/.npminstall/angular/1.5.0/angular (0 packages, use 951ms, speed 31.13kB/s, json 29.86kB, tarball 0B)
[angular-translate@2.9.2] installed at node_modules/.npminstall/angular-translate/2.9.2/angular-translate (0 packages, use 908ms, speed 30.88kB/s, json 29.86kB, tarball 0B)
[angular-loading-bar@0.8.0] installed at node_modules/.npminstall/angular-loading-bar/0.8.0/angular-loading-bar (0 packages, use 924ms, speed 30.5kB/s, json 29.86kB, tarball 0B)
[angular-ui-router-anim-in-out@1.0.2] installed at node_modules/.npminstall/angular-ui-router-anim-in-out/1.0.2/angular-ui-router-anim-in-out (0 packages, use 999ms, speed 31.37kB/s, json 33.38kB, tarball 0B)
[font-awesome@4.5.0] installed at node_modules/.npminstall/font-awesome/4.5.0/font-awesome (0 packages, use 3s, speed 54.61kB/s, json 167.38kB, tarball 0B)
[connect@3.4.1] installed at node_modules/.npminstall/connect/3.4.1/connect (0 packages, use 2s, speed 61.23kB/s, json 198.51kB, tarball 0B)
[bootstrap@3.3.6] installed at node_modules/.npminstall/bootstrap/3.3.6/bootstrap (0 packages, use 4s, speed 78.51kB/s, json 355.59kB, tarball 0B)
[graceful-fs@3.0.8] deprecate: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
[del@2.2.0] installed at node_modules/.npminstall/del/2.2.0/del (0 packages, use 3s, speed 96.34kB/s, json 762.9kB, tarball 0B)
[npmconf@2.1.1] deprecate: this package has been reintegrated into npm and is now out of date with respect to npm
[angular-ui-bootstrap@1.2.0] installed at node_modules/.npminstall/angular-ui-bootstrap/1.2.0/angular-ui-bootstrap (0 packages, use 10s, speed 94.7kB/s, json 928.5kB, tarball 0B)
[git-semver-tags@1.1.2] existed at node_modules/.npminstall/git-semver-tags/1.1.2/git-semver-tags (0 packages, use 185ms, speed 94.26kB/s, json 941.78kB, tarball 0B)
install gulp from git git+https://git@github.com/gulpjs/gulp.git#4.0, may be very slow, please keep patience
git clone -b 4.0 https://git@github.com/gulpjs/gulp.git /home/kevin/Bureau/web-template/node_modules/.npminstall/.tmp/da7bc14b-b2ba-4527-8605-d25a9272934d --depth=1
Clonage dans '/home/kevin/Bureau/web-template/node_modules/.npminstall/.tmp/da7bc14b-b2ba-4527-8605-d25a9272934d'...
[angular-ui-router@1.0.0-alpha0] installed at node_modules/.npminstall/angular-ui-router/1.0.0-alpha0/angular-ui-router (0 packages, use 10s, speed 92.57kB/s, json 954.8kB, tarball 0B)
[phantomjs@1.9.19] scripts.install: "node install.js"
[ghooks@1.0.3] scripts.install: "node ./bin/install"
[ghooks@1.0.3] installed at node_modules/.npminstall/ghooks/1.0.3/ghooks (0 packages, use 3s, speed 89.34kB/s, json 997.93kB, tarball 0B)
Looks like an `npm install -g`; unable to check for already installed version.
Download already available at /tmp/phantomjs/phantomjs-1.9.8-linux-x86_64.tar.bz2
remote: Counting objects: 69, done.
remote: Compressing objects: 100% (62/62), done.
remote: Total 69 (delta 4), reused 20 (delta 0), pack-reused 0
Dépaquetage des objets: 100% (69/69), fait.
Vérification de la connectivité... fait.
[rx@4.0.8] installed at node_modules/.npminstall/rx/4.0.8/rx (0 packages, use 11s, speed 85kB/s, json 1012.31kB, tarball 0B)
[rx-angular@1.1.3] installed at node_modules/.npminstall/rx-angular/1.1.3/rx-angular (0 packages, use 11s, speed 84.95kB/s, json 1012.31kB, tarball 0B)
[gulp-babel@6.1.2] installed at node_modules/.npminstall/gulp-babel/6.1.2/gulp-babel (0 packages, use 452ms, speed 83.71kB/s, json 1.01MB, tarball 0B)
[gulp-bump@1.0.0] installed at node_modules/.npminstall/gulp-bump/1.0.0/gulp-bump (0 packages, use 2s, speed 86.04kB/s, json 1.24MB, tarball 0B)
[conventional-github-releaser@1.1.0] installed at node_modules/.npminstall/conventional-github-releaser/1.1.0/conventional-github-releaser (0 packages, use 16s, speed 75.82kB/s, json 1.38MB, tarball 0B)
[babel-runtime@6.5.0] installed at node_modules/.npminstall/babel-runtime/6.5.0/babel-runtime (0 packages, use 18s, speed 75.3kB/s, json 1.38MB, tarball 0B)
[gulp-autoprefixer@3.1.0] installed at node_modules/.npminstall/gulp-autoprefixer/3.1.0/gulp-autoprefixer (0 packages, use 8s, speed 74.59kB/s, json 1.38MB, tarball 0B)
[babel-core@6.5.2] installed at node_modules/.npminstall/babel-core/6.5.2/babel-core (0 packages, use 18s, speed 73.9kB/s, json 1.38MB, tarball 0B)
[babel-preset-es2015@6.5.0] installed at node_modules/.npminstall/babel-preset-es2015/6.5.0/babel-preset-es2015 (0 packages, use 18s, speed 73.88kB/s, json 1.38MB, tarball 0B)
[gulp-angular-embed-templates@2.2.0] installed at node_modules/.npminstall/gulp-angular-embed-templates/2.2.0/gulp-angular-embed-templates (0 packages, use 10s, speed 69.27kB/s, json 1.39MB, tarball 0B)
[gulp-complexity@0.3.2] installed at node_modules/.npminstall/gulp-complexity/0.3.2/gulp-complexity (0 packages, use 6s, speed 69.17kB/s, json 1.39MB, tarball 0B)
[gulp-flatten@0.2.0] installed at node_modules/.npminstall/gulp-flatten/0.2.0/gulp-flatten (0 packages, use 59ms, speed 69.27kB/s, json 1.39MB, tarball 0B)
[fsevents@1.0.8] scripts.install: "node-pre-gyp install --fallback-to-build"
Verified checksum of previously downloaded file
Extracting tar contents (via spawned process)
[gulp-conventional-changelog@1.1.0] installed at node_modules/.npminstall/gulp-conventional-changelog/1.1.0/gulp-conventional-changelog (0 packages, use 2s, speed 70.61kB/s, json 1.49MB, tarball 0B)
node-pre-gyp http GET https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.0.8/fse-v1.0.8-node-v47-linux-x64.tar.gz
[gulp-concat@2.6.0] installed at node_modules/.npminstall/gulp-concat/2.6.0/gulp-concat (0 packages, use 3s, speed 70.61kB/s, json 1.51MB, tarball 0B)
[gulp-concat-css@2.2.0] installed at node_modules/.npminstall/gulp-concat-css/2.2.0/gulp-concat-css (0 packages, use 3s, speed 70.46kB/s, json 1.51MB, tarball 0B)
[gulp-git@1.7.0] installed at node_modules/.npminstall/gulp-git/1.7.0/gulp-git (0 packages, use 2s, speed 69.87kB/s, json 1.52MB, tarball 0B)
[gulp-browserify@0.5.1] installed at node_modules/.npminstall/gulp-browserify/0.5.1/gulp-browserify (0 packages, use 10s, speed 69.85kB/s, json 1.52MB, tarball 0B)
[gulp-inject@3.0.0] installed at node_modules/.npminstall/gulp-inject/3.0.0/gulp-inject (0 packages, use 1s, speed 68.41kB/s, json 1.56MB, tarball 0B)
[gulp-gzip@1.2.0] installed at node_modules/.npminstall/gulp-gzip/1.2.0/gulp-gzip (0 packages, use 2s, speed 68.36kB/s, json 1.58MB, tarball 0B)
[gulp-gh-pages@0.5.4] installed at node_modules/.npminstall/gulp-gh-pages/0.5.4/gulp-gh-pages (0 packages, use 3s, speed 69.69kB/s, json 1.64MB, tarball 0B)
[gulp-load-plugins@1.2.0] installed at node_modules/.npminstall/gulp-load-plugins/1.2.0/gulp-load-plugins (0 packages, use 1s, speed 71.08kB/s, json 1.73MB, tarball 0B)
[gulp-cssnano@2.1.1] installed at node_modules/.npminstall/gulp-cssnano/2.1.1/gulp-cssnano (0 packages, use 6s, speed 70.88kB/s, json 1.74MB, tarball 0B)
[optipng-bin@3.1.0] scripts.postinstall: "node lib/install.js"
[gulp-htmlmin@1.3.0] installed at node_modules/.npminstall/gulp-htmlmin/1.3.0/gulp-htmlmin (0 packages, use 4s, speed 71.21kB/s, json 1.77MB, tarball 0B)
module.js:341
    throw err;
    ^

Error: Cannot find module 'exec-series'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/kevin/Bureau/web-template/node_modules/.npminstall/bin-build/2.2.0/bin-build/index.js:4:18)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
[optipng-bin@3.1.0] install error: Error: Exit code 1, parentDir: /home/kevin/Bureau/web-template/node_modules/.npminstall/imagemin-optipng/4.3.0/imagemin-optipng, remove /home/kevin/Bureau/web-template/node_modules/.npminstall/optipng-bin/3.1.0/optipng-bin/.tnpminstall.done
error stack: Error: Exit code 1
    at ChildProcess.<anonymous> (/home/kevin/npm/lib/node_modules/npminstall/node_modules/runscript/index.js:38:23)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:827:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
[imagemin-optipng@4.3.0] install error: Error: Exit code 1, parentDir: /home/kevin/Bureau/web-template/node_modules/.npminstall/imagemin/4.0.0/imagemin, remove /home/kevin/Bureau/web-template/node_modules/.npminstall/imagemin-optipng/4.3.0/imagemin-optipng/.tnpminstall.done
error stack: Error: Exit code 1
    at ChildProcess.<anonymous> (/home/kevin/npm/lib/node_modules/npminstall/node_modules/runscript/index.js:38:23)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:827:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
[imagemin-optipng@^4.0.0] optional install error:
Error: Exit code 1
    at ChildProcess.<anonymous> (/home/kevin/npm/lib/node_modules/npminstall/node_modules/runscript/index.js:38:23)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:827:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
[gulp-connect@2.3.1] installed at node_modules/.npminstall/gulp-connect/2.3.1/gulp-connect (0 packages, use 7s, speed 71.07kB/s, json 1.78MB, tarball 0B)
[gulp-plumber@1.1.0] installed at node_modules/.npminstall/gulp-plumber/1.1.0/gulp-plumber (0 packages, use 324ms, speed 71.46kB/s, json 1.8MB, tarball 0B)
[gulp-rename@1.2.2] existed at node_modules/.npminstall/gulp-rename/1.2.2/gulp-rename (0 packages, use 213ms, speed 72.45kB/s, json 1.84MB, tarball 0B)
[jpegtran-bin@3.0.6] scripts.postinstall: "node lib/install.js"
[lodash@2.4.1] deprecate: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
[gifsicle@3.0.3] scripts.postinstall: "node lib/install.js"
[gulp-replace@0.5.4] installed at node_modules/.npminstall/gulp-replace/0.5.4/gulp-replace (0 packages, use 1s, speed 72.75kB/s, json 1.95MB, tarball 0B)
Removing /home/kevin/Bureau/web-template/node_modules/.npminstall/phantomjs/1.9.19/phantomjs/lib/phantom
[gulp-ng-annotate@2.0.0] installed at node_modules/.npminstall/gulp-ng-annotate/2.0.0/gulp-ng-annotate (0 packages, use 4s, speed 72.6kB/s, json 1.96MB, tarball 0B)
Copying extracted folder /tmp/phantomjs/phantomjs-1.9.8-linux-x86_64.tar.bz2-extract-1456815308726/phantomjs-1.9.8-linux-x86_64 -> /home/kevin/Bureau/web-template/node_modules/.npminstall/phantomjs/1.9.19/phantomjs/lib/phantom
Writing location.js file
Done. Phantomjs binary available at /home/kevin/Bureau/web-template/node_modules/.npminstall/phantomjs/1.9.19/phantomjs/lib/phantom/bin/phantomjs
[critical@0.7.0] installed at node_modules/.npminstall/critical/0.7.0/critical (0 packages, use 25s, speed 72.21kB/s, json 1.99MB, tarball 0B)
[gulp-rev@7.0.0] installed at node_modules/.npminstall/gulp-rev/7.0.0/gulp-rev (0 packages, use 973ms, speed 72.06kB/s, json 2MB, tarball 0B)
[lodash@1.0.2] deprecate: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
[gulp-preprocess@2.0.0] installed at node_modules/.npminstall/gulp-preprocess/2.0.0/gulp-preprocess (0 packages, use 3s, speed 72.01kB/s, json 2MB, tarball 0B)
[gulp-series@1.0.2] installed at node_modules/.npminstall/gulp-series/1.0.2/gulp-series (0 packages, use 330ms, speed 71.47kB/s, json 2MB, tarball 0B)
[graceful-fs@1.2.3] deprecate: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
[gulp-sourcemaps@1.6.0] existed at node_modules/.npminstall/gulp-sourcemaps/1.6.0/gulp-sourcemaps (0 packages, use 282ms, speed 71.09kB/s, json 2.01MB, tarball 0B)
[gulp-if@2.0.0] installed at node_modules/.npminstall/gulp-if/2.0.0/gulp-if (0 packages, use 7s, speed 71.03kB/s, json 2.02MB, tarball 0B)
[gulp-ngdocs@0.2.13] installed at node_modules/.npminstall/gulp-ngdocs/0.2.13/gulp-ngdocs (0 packages, use 4s, speed 70.74kB/s, json 2.02MB, tarball 0B)
node-pre-gyp http 404 https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.0.8/fse-v1.0.8-node-v47-linux-x64.tar.gz
node-pre-gyp http Pre-built binary not available for your system, looked for https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.0.8/fse-v1.0.8-node-v47-linux-x64.tar.gz (falling back to source compile with node-gyp) 
  ✔ gifsicle pre-build test passed successfully
[gulp-size@2.0.0] installed at node_modules/.npminstall/gulp-size/2.0.0/gulp-size (0 packages, use 1s, speed 70.28kB/s, json 2.05MB, tarball 0B)
[gulp-uglify@1.5.3] installed at node_modules/.npminstall/gulp-uglify/1.5.3/gulp-uglify (0 packages, use 627ms, speed 70.19kB/s, json 2.05MB, tarball 0B)
[gulp-notify@2.2.0] installed at node_modules/.npminstall/gulp-notify/2.2.0/gulp-notify (0 packages, use 5s, speed 70.18kB/s, json 2.05MB, tarball 0B)
[npmconf@2.1.2] deprecate: this package has been reintegrated into npm and is now out of date with respect to npm
[gulp-strip-comments@2.2.1] installed at node_modules/.npminstall/gulp-strip-comments/2.2.1/gulp-strip-comments (0 packages, use 973ms, speed 70.1kB/s, json 2.05MB, tarball 0B)
[gulp-util@3.0.7] existed at node_modules/.npminstall/gulp-util/3.0.7/gulp-util (0 packages, use 743ms, speed 69.8kB/s, json 2.09MB, tarball 0B)
make: Entering directory '/home/kevin/Bureau/web-template/node_modules/.npminstall/fsevents/1.0.8/fsevents/build'
  SOLINK_MODULE(target) Release/obj.target/.node
[spawn-sync@1.0.15] scripts.postinstall: "node postinstall"
  COPY Release/.node
make: Leaving directory '/home/kevin/Bureau/web-template/node_modules/.npminstall/fsevents/1.0.8/fsevents/build'
[gulp@https://git@github.com/gulpjs/gulp.git#4.0] installed at node_modules/.npminstall/gulp/4.0.0-alpha.2/gulp (0 packages, use 21s, speed 70.51kB/s, json 2.16MB, tarball 0B)
[gulp-karma-runner@1.1.9] installed at node_modules/.npminstall/gulp-karma-runner/1.1.9/gulp-karma-runner (0 packages, use 8s, speed 70.49kB/s, json 2.16MB, tarball 0B)
[gulp-zip@3.2.0] installed at node_modules/.npminstall/gulp-zip/3.2.0/gulp-zip (0 packages, use 1s, speed 70.39kB/s, json 2.16MB, tarball 0B)
[pngquant-bin@3.1.0] scripts.postinstall: "node lib/install.js"
[install@0.5.1] installed at node_modules/.npminstall/install/0.5.1/install (0 packages, use 867ms, speed 70.67kB/s, json 2.22MB, tarball 0B)
[gulp-strip-debug@1.1.0] installed at node_modules/.npminstall/gulp-strip-debug/1.1.0/gulp-strip-debug (0 packages, use 3s, speed 70.76kB/s, json 2.24MB, tarball 0B)
[node-sass@3.4.2] scripts.install: "node scripts/install.js"
[jpegtran-bin@3.0.6] existed at node_modules/.npminstall/jpegtran-bin/3.0.6/jpegtran-bin (0 packages, use 147ms, speed 70.7kB/s, json 2.25MB, tarball 0B)
[jasmine-core@2.4.1] installed at node_modules/.npminstall/jasmine-core/2.4.1/jasmine-core (0 packages, use 1s, speed 70.76kB/s, json 2.25MB, tarball 0B)
[jasmine@2.4.1] installed at node_modules/.npminstall/jasmine/2.4.1/jasmine (0 packages, use 1s, speed 70.75kB/s, json 2.25MB, tarball 0B)
[jsonfile@2.2.3] existed at node_modules/.npminstall/jsonfile/2.2.3/jsonfile (0 packages, use 408ms, speed 70.9kB/s, json 2.29MB, tarball 0B)
[jpegtran-bin@3.0.4] scripts.postinstall: "node lib/install.js"
[karma-jasmine@0.3.6] installed at node_modules/.npminstall/karma-jasmine/0.3.6/karma-jasmine (0 packages, use 277ms, speed 70.92kB/s, json 2.3MB, tarball 0B)
[karma-phantomjs-launcher@1.0.0] installed at node_modules/.npminstall/karma-phantomjs-launcher/1.0.0/karma-phantomjs-launcher (0 packages, use 656ms, speed 70.99kB/s, json 2.35MB, tarball 0B)
  ✔ pngquant pre-build test passed successfully
[imagemin-pngquant@4.2.2] installed at node_modules/.npminstall/imagemin-pngquant/4.2.2/imagemin-pngquant (0 packages, use 4s, speed 70.84kB/s, json 2.36MB, tarball 0B)
[karma@0.13.19] installed at node_modules/.npminstall/karma/0.13.19/karma (0 packages, use 2s, speed 70.84kB/s, json 2.36MB, tarball 0B)
[gulp-sass-lint@1.1.1] installed at node_modules/.npminstall/gulp-sass-lint/1.1.1/gulp-sass-lint (0 packages, use 6s, speed 70.77kB/s, json 2.36MB, tarball 0B)
[gulp-uncss@1.0.4] installed at node_modules/.npminstall/gulp-uncss/1.0.4/gulp-uncss (0 packages, use 4s, speed 70.55kB/s, json 2.37MB, tarball 0B)
[node-notifier@4.5.0] existed at node_modules/.npminstall/node-notifier/4.5.0/node-notifier (0 packages, use 177ms, speed 70.69kB/s, json 2.37MB, tarball 0B)
[marked@0.3.5] existed at node_modules/.npminstall/marked/0.3.5/marked (0 packages, use 328ms, speed 70.77kB/s, json 2.38MB, tarball 0B)
[node-sass@3.4.2] existed at node_modules/.npminstall/node-sass/3.4.2/node-sass (0 packages, use 144ms, speed 70.81kB/s, json 2.38MB, tarball 0B)
[merge2@1.0.1] installed at node_modules/.npminstall/merge2/1.0.1/merge2 (0 packages, use 521ms, speed 70.47kB/s, json 2.39MB, tarball 0B)
[serve-static@1.10.2] existed at node_modules/.npminstall/serve-static/1.10.2/serve-static (0 packages, use 142ms, speed 70.63kB/s, json 2.4MB, tarball 0B)
[opn@4.0.0] installed at node_modules/.npminstall/opn/4.0.0/opn (0 packages, use 538ms, speed 70.69kB/s, json 2.41MB, tarball 0B)
[gulp-xo@0.8.0] installed at node_modules/.npminstall/gulp-xo/0.8.0/gulp-xo (0 packages, use 5s, speed 70.5kB/s, json 2.42MB, tarball 0B)
[karma-coverage@0.5.3] installed at node_modules/.npminstall/karma-coverage/0.5.3/karma-coverage (0 packages, use 2s, speed 70.42kB/s, json 2.42MB, tarball 0B)
[xo@0.12.1] existed at node_modules/.npminstall/xo/0.12.1/xo (0 packages, use 281ms, speed 70.34kB/s, json 2.42MB, tarball 0B)
[path@0.12.7] installed at node_modules/.npminstall/path/0.12.7/path (0 packages, use 756ms, speed 70.3kB/s, json 2.42MB, tarball 0B)
  ✔ jpegtran pre-build test passed successfully
[jpegtran@0.2.0] installed at node_modules/.npminstall/jpegtran/0.2.0/jpegtran (0 packages, use 3s, speed 69.94kB/s, json 2.42MB, tarball 0B)
[validate-commit-msg@2.1.0] installed at node_modules/.npminstall/validate-commit-msg/2.1.0/validate-commit-msg (0 packages, use 748ms, speed 69.73kB/s, json 2.42MB, tarball 0B)
  ✔ jpegtran pre-build test passed successfully
[gulp-imagemin@2.4.0] installed at node_modules/.npminstall/gulp-imagemin/2.4.0/gulp-imagemin (0 packages, use 13s, speed 69.69kB/s, json 2.42MB, tarball 0B)
[yargs@4.2.0] installed at node_modules/.npminstall/yargs/4.2.0/yargs (0 packages, use 1s, speed 69.16kB/s, json 2.45MB, tarball 0B)
[less@2.6.0] installed at node_modules/.npminstall/less/2.6.0/less (0 packages, use 3s, speed 69.08kB/s, json 2.47MB, tarball 0B)
[graceful-fs@1.1.14] deprecate: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
[plato@1.5.0] installed at node_modules/.npminstall/plato/1.5.0/plato (0 packages, use 2s, speed 68.96kB/s, json 2.47MB, tarball 0B)
Binary downloaded and installed at /home/kevin/Bureau/web-template/node_modules/.npminstall/node-sass/3.4.2/node-sass/vendor/linux-x64-47/binding.node
[node-sass@3.4.2] scripts.postinstall: "node scripts/build.js"
` /home/kevin/Bureau/web-template/node_modules/.npminstall/node-sass/3.4.2/node-sass/vendor/linux-x64-47/binding.node ` exists. 
 testing binary.
Binary is fine; exiting.
[gulp-sass@2.2.0] installed at node_modules/.npminstall/gulp-sass/2.2.0/gulp-sass (0 packages, use 28s, speed 45.81kB/s, json 2.47MB, tarball 0B)
peerDependencies WARNING [karma-phantomjs-launcher@1.0.0] in /home/kevin/Bureau/web-template/node_modules/.npminstall/karma-phantomjs-launcher/1.0.0/karma-phantomjs-launcher requires a peer of phantomjs-prebuilt@>=1.9 but none was installed
All packages installed (1512 packages installed from npm registry, 1 packages installed from git, use 56s, speed 45.52kB/s, json 1869(2.47MB), tarball 0B)
kevin@kevin-Lenovo-Flex-2-14:~/Bureau/web-template$ npminstall --version
v1.1.1
```
